packages/nc-gui-v2/components/smartsheet-toolbar/SharedViewList.vue

```diff
@@ -60,7 +60,7 @@ const sharedViewUrl = (view: SharedViewType) => {
     default:
       viewType = 'view'
   }
-  return `/nc/${viewType}/${view.uuid}`
+  return `nc/${viewType}/${view.uuid}`
```

Comment:
You can combine `${dashboardUrl?.value as string}/....` here

Reply:
@pranavxc check if the changes are as you had intended.